Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.yml

```diff
@@ -1301,10 +1304,64 @@ script:
     - contextPath: CarbonBlackEEDR.SearchProcess.results.scriptload_count
       description: The cumulative count of loaded scripts since process tracking started.
       type: Number
-  dockerimage: demisto/python3:3.10.13.72123
+  - arguments:
+    - description: Threat ID to add the tags to.
+      isArray: true
+      name: threat_id
+      required: true
+    - description: The tags to be added to the threat.
+      name: tags
+      required: true
```

Comment:
Shouldn't this be a list as well?
If so let's add the "Comma-separated list of..." to the description and the isArray: true.

Reply:
Fixed, threat is string, tags is list.